### DISCUSSION TITLE
Separate Regional Carbon Markets Subject to Forcing Target

### DISCUSSION THREAD

**Discussion by BmarcyB** [JGCRI/gcam-core#299] [Q&A]
Hey - potentially dumb question but please bear with me!

I am trying to set up a policy forcing target (forcing_target_4p5 for example), but allow each region to solve for their own carbon price, to get regional carbon prices. 

How do I do this? I have tried setting up different linked policies, but no luck.

Any help would be most appreciated!

Thank you!

**Comment by pkyle** [comment]
As far as I know, to get regionally differentiated carbon prices one would need regionally differentiated emissions/climate targets. Because 4.5 is a global target, the model solves for the global GHG price to meet the target. You could run the 4.5 to see what the global CO2 budget is, and then read in region-specific CO2 emissions constraints that add up to the global total but that are different than what was observed in the scenario. Because of non-CO2 forcing this approach wouldn't return exactly 4.5.
